src/general/nqueens.rs

```diff
@@ -0,0 +1,153 @@
+#[allow(unused_imports)]
+use std::env::args;
+
+#[allow(dead_code)]
```

Comment:
I am not sure why it's needed, and `unused_imports` as well.

Reply:
I included it because it was needed:

`#[allow(unused_imports)]` was added because `cargo test` reports that `std::env::args` is unused.

`#[allow(dead_code)]` was added because `cargo test` again reports that `fn main()` and `fn print_board()` are never used.

I couldn't find a way around those compiler flags without taking features out of the program.